templates/credential-registry/cargo-generate.toml

```diff
@@ -0,0 +1,17 @@
+[template]
+cargo_generate_version = ">= 0.16.0, < 0.19.0"
```

Comment:
In the CI file you say that this needs at least 0.17.5. Shouldn't that be reflected here?

Reply:
Done, bumped the minimal supported version to 0.17.0 (commented on why the CI uses 0.17.5)